sklearn/model_selection/_split.py

```diff
@@ -1226,9 +1246,11 @@ class RepeatedKFold(_RepeatedSplits):
     --------
     RepeatedStratifiedKFold: Repeats Stratified K-Fold n times.
     """
-    def __init__(self, n_splits=5, n_repeats=10, random_state=None):
+    @_deprecate_positional_args
+    def __init__(self, n_splits=5, *, n_repeats=10, random_state=None):
```

Comment:
do we want to include n_splits also? In `RepeatedKFolds(4)`we don't know what 4 refers to.

Reply:
Agreed
```suggestion
    def __init__(self, *, n_splits=5, n_repeats=10, random_state=None):
```

Although we do force users to a more verbose `RepeatKFold(n_splits=n_splits, ...)`